CHANGELOG.txt

```diff
@@ -1,3 +1,9 @@
+=== 3.4.0 (unreleased) ===
+
+* Changed CMS JavaScript bundling from simple concatenation to webpack-based.
```

Comment:
we can, however it would require some coordination on addons (e.g. djangocms-text-ckeditor needs to be released and pinned in appropriate places)